### PR TITLE
New 526 Wizard

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Changes up the disability form finding wizard according to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/17306.

The previous wizard was pretty hacked together, so I decided to take a crack at something perhaps a little cleaner. This is what I came up with. I tried to strike a balance between reusability, customization, and ease of development.

**Note:** I think there are two main deviations between this wizard and the previous one:
1) It doesn't check if the user is signed in or authenticated for the app (has `evss-claims` in their services list)
2) It doesn't require that an option be selected so much as it just disables the button until one is selected
   - The date input still says to enter a valid date, but if the year is wonky (`1019`, for instance), it'll give them a validation message but will still let them hit next

## Testing done
Lotsa manual testing.

## Screenshots
image
**No** :point_down: 
image
Should we plug in the dates for the 180 and 90 days in the future?
**Yes** :point_down: 
image
**I'm filing a claim...** :point_down: 
image
@goldenmeanie @katelyncai @peggygannon This seems...like it needs more, yeah?
**I'm disagreeing** :point_down: 
image
**Date more recent than Feb 19, 2019** :point_down: 
image
**Date more recent than 1 year ago** :point_down: 
image
**Date over 1 year ago** :point_down: 
image

## Acceptance criteria
- [x] All the pages show when they're supposed to

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by peggygannon — CHANGES_REQUESTED** [review]
Hi, @cvalarida @goldenmeanie @katelyncai . here are some additional content recommendations for the wizard. Let me know your thoughts? 
Thanks! 

1. Looking at the wizard laid out, I think we need to add a line letting users know why we're asking a series of questions. I recommend adding this line: ADD: Just answer a few questions, and we’ll show you where to file your disability claim. 

so it would read: 
**How do I file my claim?**
You can file online right now. Just answer a few questions, and we'll show you where to file your disability claim. 
{button} File a Disability Compensation Claim (expands to wizard) 

2. Add a question about first time filing claim
{If Yes to separated question}
ADD: Is this your first time filing a claim with VA? 
{if yes: }
**You'll need to file a claim on eBenefits**
To file your first disability claim, please go to our eBenefits website. 
{button} Go to eBenefits

{if no:} 
Are you filing a new disability claim or are you disagreeing with VA on an **earlier** claim?  **please change prior to earlier** in this question

3. {If yes to I'm filing a new claim} let's update message to: 
To file your disability claim, please sign in and verify your account.
{button} Sign in to Start Your Application

**Comment by cvalarida** [comment]
@markgreenburg @goldenmeanie and I met earlier today and I think we may have more changes to the original claims section.

@goldenmeanie @markgreenburg Your thoughts?

**Comment by cvalarida** [comment]
Per @goldenmeanie's guidance, I've rejiggered the wizard to show all the "pages" as a stack rather than showing only one at a time.

Some updated screenshots:
image
image

The only content changes have been what @peggygannon requested above.

File: src/applications/disability-benefits/wizard/pages/disagreeing.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// Figure out which page to go to based on the date entered
+const findNextPage = state => {
+  const date = moment({
+    day: state.day.value,
+    // moment takes 0-indexed months, but the date picker provides 1-indexed months
+    month: parseInt(state.month.value, 10) - 1,
+    year: state.year.value,
+  });
+  const feb19 = moment('2019-02-19');
+
```
Comment (bshyong) [inline src/applications/disability-benefits/wizard/pages/disagreeing.jsx]: Saw this magic date in the related issue; just double checking that we won’t need to change this ever?

We may also want to leave a comment for anybody else looking at this in the future

**Review by bshyong — APPROVED** [review]
:100:

**Comment by peggygannon** [comment]
thanks, @cvalarida my only edit for this recent update is: 

please change "a" to "an" before earlier in this sentence: 
Are you filing a new disability claim or are you disagreeing with VA on an earlier claim?

**Review by goldenmeanie — CHANGES_REQUESTED** [review]
- Ultimate button should be green
- Check text-wrap on radio buttons to make sure they are rendering bulleted-list-style
- Should use alert styling for the appeal notification content

File: src/applications/disability-benefits/wizard/pages/disagreeing.jsx (hunk 1 above)
Comment (cvalarida) [reply src/applications/disability-benefits/wizard/pages/disagreeing.jsx]: Good call. I think it's pretty much set in stone; I'll drop a comment in there for clarification.

**Comment by cvalarida** [comment]
# Updated screenshots
image
image
image
image
image
image

**Comment by cvalarida** [comment]
@goldenmeanie The radio button thing is in `formation-react`. We can write a ticket for that, but should it block this PR?

**Comment by goldenmeanie** [comment]
> @goldenmeanie The radio button thing is in formation-react. We can write a ticket for that, but should it block this PR?

Not a blocker

**Comment by goldenmeanie** [comment]
Also not a blocker, but we will need to modify the older than a year content. Following up with Paul, but apparently the action is different than submit a 526EZ (equivalent)

**Comment by peggygannon** [comment]
thanks, @cvalarida A couple of content changes that still need to be updated. Updates are in bold, just for emphasis here: 

1. please update first claim question to: Is this your first time filing a **disability** claim with VA?

2. please change "a" to "an" before earlier in this sentence:
Are you filing a new disability claim or are you disagreeing with VA on **an** earlier claim?

3. For the older than a year  alert box, please tweak message to 
If you disagree with a VA decision that you received more than a year ago, you’ll need to file a new **disability** claim.

3. for the decision review alert box, please remove the period from the bold header, so it should be 
You'll need to request a decision review

**Comment by cvalarida** [comment]
Consarn it. I could have sworn I put those in there already...

:ghost: I'll get it fixed up.

**Comment by cvalarida** [comment]
Ahh, turns out I _did_ have those edits in already—except for the last two alert ones, that is.

Updated screenshots:
image

image

**Comment by peggygannon** [comment]
Hi, @cvalarida  these recent updates look good. 

As I mentioned in Slack Alex and I are confirming what the message should be for the claims for more than 1 year ...

**Comment by peggygannon** [comment]
@goldenmeanie @cvalarida Please see updated message below for appeals more than 1 year ago. Let me know if you have any questions: 

{If more than a year}
**You'll need to file a Supplemental Claim**
If you disagree with a VA decision that you received more than a year ago, you'll need to file a Supplemental Claim. To file a Supplemental Claim, fill out the Decision Review Request: Supplemental Claim (VA Form 20-0995).  
[Download VA Form 20-0995](www.va.gov/decision-reviews/forms/supplemental-claim-20-0995.pdf)

[Learn more about Supplemental Claims](decision-reviews/supplemental-claim/)

**Comment by goldenmeanie** [comment]
In this case they have already told us that they are disagreeing. Does it seem reasonable to do this?

> Since you received your VA decision more than a year ago, you'll need to file a Supplemental Claim. Fill out the Decision Review Request: Supplemental Claim (VA Form 20-0995).

**Comment by cvalarida** [comment]
@peggygannon @goldenmeanie How does this look?
image

**Comment by peggygannon** [comment]
@cvalarida. We just need to update the bold header to: 
You'll need to file a Supplemental Claim

**Comment by cvalarida** [comment]
image

**Review by peggygannon — APPROVED** [review]

**Comment by goldenmeanie** [comment]
Spoke with some folks and feel confident we need to swap the >1 year state back to the `file a new claim` alert shown here https://github.com/department-of-veterans-affairs/vets-website/pull/9726#issuecomment-474018915
@cvalarida

**Comment by cvalarida** [comment]
:man_shrugging:

**Comment by cvalarida** [comment]
Just reverted the supplemental claim commits and we're back to:
image

**Review by goldenmeanie — APPROVED** [review]

**Comment by peggygannon** [comment]
@cvalarida Looks good. thanks...

**Pull request merged**